bin/setup_dependencies.sh

```diff
@@ -37,8 +37,8 @@ chmod -R 600 /home/ubuntu/.ssh
 chmod +x /home/ubuntu/.ssh
 
 # Disable root access
-passwd -l root
-echo 'SSH access for root disabled. You will need to connect as ubuntu.'
+# passwd -l root
```

Comment:
I'd prefer not to have this in there @carande although I can understand that this may be a pain point for you.